packages/notification-service/src/polling.ts

```diff
@@ -13,3 +14,14 @@ export const notificationPolling = AsyncPolling(async (end) => {
     end()
   }
 }, POLLING_INTERVAL)
+
+export const exchangePolling = AsyncPolling(async (end) => {
+  try {
+    const web3 = await getWeb3Instance()
```

Comment:
Nit, any reason to get web3 here given that it's in the same file as handleExchangeQuery?

Reply:
Fixed